icu4c/source/test/intltest/units_test.cpp

```diff
@@ -1214,16 +1216,18 @@ void UnitsTest::testUnitsConstantsDenomenator() {
         {"portion-per-1000000", 1000000},
         {"portion-per-10000000", 10000000},
         {"portion-per-100000000", 100000000},
-        // ICU-22781: {"portion-per-1000000000", 1000000000},
-        // ICU-22781: {"portion-per-10000000000", 10000000000},
-        // ICU-22781: {"portion-per-100000000000", 100000000000},
-        // ICU-22781: {"portion-per-1000000000000", 1000000000000},
-        // ICU-22781: {"portion-per-10000000000000", 10000000000000},
-        // ICU-22781: {"portion-per-100000000000000", 100000000000000},
-        // ICU-22781: {"portion-per-1000000000000000", 1000000000000000},
-        // ICU-22781: {"portion-per-10000000000000000", 10000000000000000},
-        // ICU-22781: {"portion-per-100000000000000000", 100000000000000000},
-        // ICU-22781: {"portion-per-1000000000000000000", 1000000000000000000},
+        {"portion-per-1000000000", 1000000000},
+        {"portion-per-10000000000", 10000000000},
+        {"portion-per-100000000000", 100000000000},
+        {"portion-per-1000000000000", 1000000000000},
+        {"portion-per-10000000000000", 10000000000000},
+        {"portion-per-100000000000000", 100000000000000},
+        {"portion-per-1000000000000000", 1000000000000000},
+        {"portion-per-10000000000000000", 10000000000000000},
+        {"portion-per-100000000000000000", 100000000000000000},
+        {"portion-per-1000000000000000000", 1000000000000000000},
```

Comment:
Same issue as in the other PRs: If these are all expected to fail and are being covered by `logKnownIssue()` in the code, put some sort of comment next to the actual test case to indicate that (the ticket number you had in the earlier version of these changes works, for example).  Same thing further down in here.

An alternative approach would be to add a `knownIssueTicketNumber` property to your `TestCase` object-- the test code would look for that and, if it's populated, skip the test and pass it to `logKnownIssue`.

Reply:
Is line 1219 the only one that's failing?  If so, we're good here.